internal/service/securitylake/data_lake_test.go

```diff
@@ -581,6 +652,43 @@ resource "aws_securitylake_data_lake" "test" {
 `, rName, acctest.Region()))
 }
 
+func testAccDataLakeConfig_metaStore(rName string) string {
+	return acctest.ConfigCompose(testAccDataLakeConfigConfig_base, fmt.Sprintf(`
+resource "aws_securitylake_data_lake" "test" {
+  meta_store_manager_role_arn = aws_iam_role.meta_store_manager.arn
+
+  configuration {
+    region = %[2]q
+  }
+
+  tags = {
+    Name = %[1]q
+  }
+
+  depends_on = [aws_iam_role.meta_store_manager]
```

Comment:
Since `meta_store_manager_role_arn` already refers to `aws_iam_role.meta_store_manager`, this `depends_on` is redundant.